### ISSUE TITLE
Not finding karma.yaml in root

### ISSUE THREAD

**Issue by mh720** [prymitive/karma#261]
Howdy Lukasz, 

Great work on this, I just found out about your work on Karma after CloudFlare. I'm evaluating it now to replace unsee in swarmstack (https://www.youtube.com/watch?v=3FpTcVnvfRg - https://github.com/swarmstack/swarmstack) - which rebuilt Stefan Prodan's swarmprom starter kit for Docker Prometheus/Grafana/Alertmanager/Unsee and adds Docker swarm node installation and ongoing management with ease of installation as a focus. 

Dockerfile below works, but requires addition of the CONFIG_FILE env variable. My reading of docs/CONFIGURATION.md was that Karma would look for a file named karma.yaml in the same directory. Didn't see an existing ticket on this issue, but might be pebkac as I just found out about Karma an hour ago :)

```
#cat docker-compose.yml
version: "3.4"
configs:
  karma-conf:
    file: ./karma/karma.yaml

services:
  karma:
    image: lmierzwa/karma:latest
    configs:
      - source: karma-conf
        target: /karma.yaml
    deploy:
      mode: replicated
      replicas: 1
      labels:
        prometheus.enable: "true"
        prometheus.port: "8080"
        prometheus.path: "/metrics"
    environment:
      - CONFIG_FILE=/karma.yaml
```

**Comment by prymitive** [comment]
Hi, it's always nice to hear that other finds it valuable so thanks.
The issue you're facing is definitely a bug, likely introduced with #28.
I was able to reproduce and I've raised #262 to fix it. Sorry about that.

**Comment by mh720** [comment]
Thanks, the fix worked. I've flipped https://github.com/swarmstack/swarmstack users to use my fork swarmstack/karma for the time-being. I tried to create a PR on GitHub for my changes back to you, but I'm not good with cherry-picking and it would have picked up my change to README.md which just informs users that it adds /bin/sh and /bin/wget to your image. If you want to pick up these changes, I'd prefer to delete my fork and point swarmstack at your images instead. I think you'll enjoy the size-reduction I've added to your binary using UPX to compress it:

https://github.com/swarmstack/karma/blob/master/Dockerfile
https://github.com/swarmstack/karma/blob/master/Dockerfile_healthcheck
